### PR TITLE
Add script to revert changes

### PR DESCRIPTION
**Pull request by samely**
### :scroll:  **This script allows to**:

:green_heart: Revert a delta by specifying the `id`
:green_heart: Revert all the features at once
:green_heart: Decide whether to `revert` or `keep` each feature.
:green_heart: Mix `keep/revert` with `keep all/revert all` to revert just some features

### :pencil2:  How is revertion working?
:point_right:  We have actions that could be...

image

:black_circle: _actions when making changes_  :large_blue_circle: _actions when reverting changes_

:point_right: According to the image above, this is how revert is working :point_down: 
image

_Direct relation when reverting changes_

@ingalls

### CODE REVIEW

**Comment by samely** [comment]
:heavy_plus_sign:  I've also added the option to send an array of _features id_ to be reverted.

@ingalls could you review this PR? :pray:

**Comment by ingalls** [comment]
Just playing around against a local Hecate instance, this is looking awesome, another thing

screenshot from 2018-06-05 09-46-38

If the current version is the same as the server version let's revert automatically and not prompt the user as there is no conflict.

File: lib/revert.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            getFeatureHistory(options, feature.id, (err, response) => {
+                if (err) return callback(err);
+                if (response.statusCode !== 200) return callback(new Error(`ERROR: ${response.body.reason}`));
+                if (response.body.length === 0) return callback(new Error('ERROR: History feature collection cannot be empty'));
+
+                const currentFeature = response.body[0].feat;
+
+                this.getRevertToFeature(response.body, feature, (err, revertToFeature) => {
+                    if (err) return callback(err);
+                    if (!revertToFeature) return callback(new Error('Feature to rever to could not be defined'));
```
[the remark this answers is not in the document]
Comment (samely) [reply lib/revert.js]: @ingalls what does this mean?

[the remark this answers is not in the document]
Comment (samely) [reply lib/revert.js]: oh... got it! Thanks

**Comment by ingalls** [comment]
:taco: LGTM

**Pull request merged**